src/main/java/uk/gov/ons/ctp/common/UnirestInitialiser.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.ons.ctp.common;
+
+import com.google.gson.Gson;
+import com.mashape.unirest.http.Unirest;
+
+public class UnirestInitialiser {
+  public static void initialise() {
+    final Gson gson = new Gson();
```

Comment:
Why use Gson as opposed to say Jackson mapper used absolutely everywhere else?

Reply:
The checked exception to wrap with a Runtime was annoying me and wanted to remove (and forgot) to remove the full path for `com.mashape.unirest.http.ObjectMapper` because the class name doesn't clash

Reply:
It should probably use the existing ObjectMapper, wrapping the exception kinda sucks but a new dependency doing something that's already done in another sucks more.

Reply:
Two against one, it's changed 😅